src/gf.c

```diff
@@ -2352,7 +2365,42 @@ static void record_precompile_statement(jl_method_instance_t *mi)
     if (!jl_has_free_typevars(mi->specTypes)) {
         jl_printf(s_precompile, "precompile(");
         jl_static_show(s_precompile, mi->specTypes);
-        jl_printf(s_precompile, ")\n");
+        jl_printf(s_precompile, ") # ");
+
+        jl_bt_element_t *bt = (jl_bt_element_t*)malloc(JL_MAX_BT_SIZE*sizeof(jl_bt_element_t));
```

Comment:
This is too much to malloc without crashing the program. At least use the dynamic `jl_backtrace_from_here` if you need unbounded time and space here.